### PR TITLE
Refactor: Help page

### LINKED ISSUE #315: Update Help Pages for better User Experience
Based on feedback from MST staff, changes to help pages to improve user experience

Todo:

- [ ] Review draft help page with staff (Olivia, MST)
- [x] Update Help page content with new sections / paragraphs, using existing `core/help.html` template
- [x] Make new sub-sections anchor links e.g. `#payment-options`
- [x] Remove/ delete any links to `core:payment_options`
- [x] on Footer, remove link to "Payment Options"
- [ ] on Eligibility Start page, change link "How do I know if I have a contactless card" to /help
- [ ] on Eligibility Start page, change link "what if I don't have a bank card" to /help
- [x] on Enrollment Start page, change link "What if I don't have a bank card" /help
- [x] update UI tests for Help page, incorporate UI tests for removed Payment Options

Resources

 - ~[Large_SupportingPage_Help V2](https://www.figma.com/file/5IabSdY0xDzwL4gyXhbQ9G/Benefits-(Courtesy-Cards)?node-id=2004%3A6408)~
- [Desktop Help](https://www.figma.com/file/5IabSdY0xDzwL4gyXhbQ9G/Benefits-(Courtesy-Cards)?node-id=2148%3A10895)

### PR DESCRIPTION
**Pull request by angela-tran**
Closes #315

---
**Notes on changes**
 - The two bullet points on #315 that mention the Eligibility start page will be handled by #365.
 - Removed all links to old Payment Options page.
 - Updated enrollment start page to point to help page. 
 - Added new UI tests for Help page.
    - To run just the help tests, you can do `npx cypress run --spec "specs/ui/help.spec.js"` (from the `tests/cypress` directory)

Here is what the page currently looks like:
Screenshot 2022-04-07 at 17-09-49 Transit Benefits Help Cal-ITP

### CODE REVIEW

**Review by machikoyasuda — DISMISSED** [review]
The use of `blocktranslate` is very cool!

File: benefits/enrollment/views.py [hunk 1]
```diff
@@ -36,9 +36,7 @@ def _index(request):
             viewmodels.Button.primary(
                 text=_("enrollment.buttons.payment_partner"), id=tokenize_button, url=f"#{tokenize_button}"
             ),
-            viewmodels.Button.link(
-                classes="btn-sm", text=_("enrollment.buttons.payment_options"), url=reverse("core:payment_options")
-            ),
+            viewmodels.Button.link(classes="btn-sm", text=_("enrollment.buttons.payment_options"), url=reverse("core:help")),
```
Comment (thekaveman) [inline benefits/enrollment/views.py:39]: Is there a way to link get a link to the `#payment-options` header with `reverse()`?

Comment (angela-tran) [reply benefits/enrollment/views.py]: I didn't see anything about doing it with `reverse`, but we can append the anchor to the string returned from `reverse`. I tried out this snippet:
```python
     viewmodels.Button.link(
           classes="btn-sm", text=_("enrollment.buttons.payment_options"), url=f"{help_page}#payment-options"
     ),
```
We get the anchor!
image

Weird Firefox thing I'm seeing though is that clicking the link will use the right URL, but the browser doesn't go to the anchor. If I refresh the page though, then it goes to the right place on the page. 🤔 Chrome doesn't have this weird issue.

**Comment by machikoyasuda** [comment]
Add `rel="noopener noreferrer` and `target="_blank"` to the Login.gov and LittlePay links: image

File: benefits/enrollment/views.py (hunk 1 above)
Comment (angela-tran) [reply benefits/enrollment/views.py]: @machikoyasuda tested on her Mac with Firefox, and it worked! 🤷

**Comment by machikoyasuda** [comment]
Instructions for temporary bypassing the eligibility form, for testing purposes only:

1. Hide the Debug bar

benefits/static/css/styles.css
```diff
 /* HTML element overrides */
 
+.bg-danger {
+  display: none;
+}
```

2. Bypass verification by commenting out code in `confirm()` method on `benefits/eligibility/views.py`:

```diff
     if request.method == "POST":
-        analytics.started_eligibility(request)
+        # analytics.started_eligibility(request)
 
-        form = forms.EligibilityVerificationForm(data=request.POST, verifier=verifier)
-        response = _verify(request, form)
+        # form = forms.EligibilityVerificationForm(data=request.POST, verifier=verifier)
+        # response = _verify(request, form)
 
-        if response is None:
-            # form was not valid, allow for correction/resubmission
-            analytics.returned_error(request, form.errors)
uest, form.errors)
+        #     page.forms = [form]
+        #     response = PageTemplateResponse(request, page)
+        response = verified(request, ["type2"])
     elif session.eligible(request):
         eligibility = session.eligibility(request)
         response = verified(request, [eligibility.name])
```

3. Hide the `invisible` CSS class on the `/enrollment` page:

image

**Review by machikoyasuda — APPROVED** [review]
Confirmed that links are opening in a new tab 🙌

**Pull request merged**